lib/drivers/cryptoid.js

```diff
@@ -0,0 +1,91 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+const Coin = require('../models/coin');
+
+class CryptoID extends Driver {
+  constructor(options) {
+    super({
+      timeout: 200, // 5 requests per second
+      options,
+      supports: {
+        native: false,
+        total: true,
+        circulating: true,
+        blockchains: ['Litecoin', 'Emerald', 'Dash'],
+      },
+    });
+  }
+
+  async totalSupplyLitecoin() {
+    const total = await this.request('https://chainz.cryptoid.info/ltc/api.dws?q=totalcoins');
+    return Number(total);
+  }
+
+  async totalSupplyEmerald() {
+    const total = await this.request('https://chainz.cryptoid.info/emd/api.dws?q=totalcoins');
+    return Number(total);
+  }
+
+  async totalSupplyDash() {
+    const total = await this.request('https://chainz.cryptoid.info/dash/api.dws?q=totalcoins');
+    return Number(total);
+  }
+
+  /**
+   * @augments Driver.fetchTotalSupply
+   * @param {string} blockchain one of supported blockchains
+   * @async
+   */
+  async fetchTotalSupply(blockchain) {
+    if (this.supportsBlockchain(blockchain) && this[`totalSupply${blockchain}`]) {
+      return this[`totalSupply${blockchain}`]();
+    }
+
+    return null;
```

Comment:
This way it will be the same as the BlockChair driver
```suggestion
    throw new Error(`The blockchain "${blockchain}" is not supported in this driver`);
```